ui-tests/src/test/java/io/syndesis/qe/steps/connections/wizard/phases/ConfigureConnectionSteps.java

```diff
@@ -46,11 +45,7 @@ public void fillConnectionDetails(String connectionName) {
     }
 
     private void fillFormInLowerCase(Map<String, String> properties) {
-        //transfer all keys in properties to lower case
```

Comment:
This is not longer needed since all fill form methods wait for the form to load... it's a duplication of code and the method in form works with a little change